lib/templates/android_screen_base.tt

```diff
@@ -183,4 +183,55 @@ class AndroidScreenBase < Calabash::ABase
     # Clicking in the Done button
     touch "* id:'button1'"
   end
+  
+  # @param query [String] query that will be searched
+  # @param scroll_location_id [String]
+  # @param scroll_location_query [String, nil]
+  # @param direction [Symbol]
+  # @param options [Hash]
+  # @option options [String] :timeout
+  # @option options [String] :flick in case we need flick
+  # @todo #TODO Add reset option to allow going to top position
+  # before start search
+  # Exemple -> scroll_looking_for_query(query_text_variable, nil, scroll_location_query_variable, :down, :timeout=>7)
+
+  def scroll_looking_for_query query, scroll_location_id, scroll_location_query = nil , direction= :down, options = {}
+
+    if scroll_location_query.nil?
+      scroll_location_query="* id:'#{scroll_location_id}'"
+    end
+
+    if options[:timeout].nil?
+      options[:timeout]=15
+    end
+
+    if options[:flick].nil?
```

Comment:
@oscartanner ,
my [method](https://github.com/rafaelanastacioalves/sunomono/blob/f40a6997b4ddbc513ea085f7110871bc22b784b2/lib/templates/android_screen_base.tt#L198) has two queries:
1 - query: the query we are looking for during scroll;
2 - scroll_location_query: the scrollable element. Sometimes the screen has more than one scrollable element. So I added this optional element to overcome this case.